plugins/modules/template_workflow_manager.py

```diff
@@ -1331,7 +1348,7 @@ def get_template_info(self, template_params):
             if parameter_name is not None:
                 templateParams[i].update({"parameterName": parameter_name})
             else:
-                self.msg = "parameter_name is mandatory for the template_params."
+                self.msg = "parameter_name is required for the template_params."
```

Comment:
"The parameter 'parameter_name' is required for 'template_params' but not provided."